schema/home/home_page_hero_units.js

```diff
@@ -0,0 +1,86 @@
+import cached from '../fields/cached';
+import gravity from '../../lib/loaders/gravity';
+import { GravityIDFields } from '../object_identification';
+import {
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+
+const HomePageHeroUnitType = new GraphQLObjectType({
+  name: 'HomePageHeroUnit',
+  fields: {
+    ...GravityIDFields,
+    cached,
+    heading: {
+      type: GraphQLString,
+    },
+    href: {
+      type: GraphQLString,
+      resolve: ({ link }) => link,
+    },
+    title: {
+      type: GraphQLString,
+      resolve: ({ mobile_title }) => mobile_title,
```

Comment:
so resolving makes sure that there's something there, and if there is, then send have that be the content for 'title'?